distrobox-assemble

```diff
@@ -370,15 +370,27 @@ run_distrobox() (
 		"${distrobox_path}"/distrobox enter "${name}" -- touch /dev/null
 
 		IFS="¤"
-		for app in ${exported_apps}; do
-			app="$(echo "${app}" | tr -d '[:space:]')"
-			"${distrobox_path}"/distrobox enter "${name}" -- distrobox-export --app "${app}"
+		for apps in ${exported_apps}; do
+			# Split the string by spaces
+			IFS=" "
+			for app in ${apps}; do
+				# Remove spaces
+				app="$(echo "${app}" | tr -d '[:space:]')"
```

Comment:
the script involves splitting a string by spaces and then trimming spaces from each resulting element. However, I'm curious if the trimming step is actually necessary.

I tried the following code in shell, results are identical
```sh
apps="1   2    3  4 5 6"
IFS=" "
echo with trimming
for app in ${apps}; do
        app="$(echo "${app}" | tr -d '[:space:]')"
        echo "|$app|"
done
echo without trimming
for app in ${apps}; do
        echo "|$app|"
done
```

Reply:
You're right. I removed the `tr -d '[:space:]')`.